Dockerfile

```diff
@@ -182,7 +172,15 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     && pip check \
     && find /opt/warehouse -name '*.pyc' -delete
 
+# Install our development dependencies if we're building a development install
```

Comment:
I think we don't need to change these lines anymore?

Reply:
yep, done